src/utils/db.js

```
@@ -0,0 +1,11 @@
+const { Sequelize } = require('sequelize');
+
+const client = new Sequelize({
```

Consider explicitly reading and validating required environment variables so you fail early with a clear error if something is missing. Right now Sequelize will throw later but the error may be less obvious. For example, check that `DB_HOST`, `DB_USER`, `DB_PASSWORD`, and `DB_DATABASE` are present and throw a descriptive error if not.